substrate/frame/tips/src/tests.rs

```diff
@@ -192,7 +192,7 @@ impl Config for Test {
 	type Tippers = TenToFourteen;
 	type TipCountdown = ConstU64<1>;
 	type TipFindersFee = TipFindersFee;
-	type TipReportDepositBase = ConstU64<1>;
+	type TipReportDepositBase = ConstU64<0>;
```

Comment:
Please don't change the DepositBase to 0. It's important that it is non-zero so we can ensure it works correctly in the test.

Reply:
The change to zero was needed to test the invariant theory.

Reply:
We shouldn't test one thing at the expense of something else.

If you want a tip with zero deposit for the purposes of testing this invariant, please insert it manually into storage.

Reply:
Thanks for the review, will change accordingly.